scripts/verify-bootnodes.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ $1 = "karura" ]; then
+	IMAGE=acala/karura-node:1.0.1
+	CHAIN_NAME=karura
+	GENESIS=./resources/karura-dist.json
+elif [ $1 = "acala" ]; then
+	IMAGE=acala/acala-node:1.0.1
+	CHAIN_NAME=acala
+	GENESIS=
+else
+	echo "not support $1"
+	exit 1
+fi
+
+docker rm -f bootnotes > /dev/null 2>&1 || true
```

Comment:
why need this?

Reply:
Deal with the situation of `ctrl+c` in local testing. 
Will not affect the process.